theme/templates/base.html

```diff
@@ -40,7 +46,7 @@
               <a class="icon fa-slack" href="https://slack.kinto-storage.org/" title="Slack channel">Slack</a>
             </li>
             <li>
-              <a class="icon fa-rss" href="http://www.kinto-storage.org/archives.html" title="Latest articles about Kinto">Blog</a>
+              <a class="icon fa-rss" href="./archives.html" title="Latest articles about Kinto">Blog</a>

```

Comment:
You might also point to `news.html` once you've restored this behaviour (not sure why you did this change).